crates/astria-sequencer/src/bridge/bridge_lock_action.rs

```diff
@@ -145,22 +149,17 @@ impl ActionHandler for BridgeLockAction {
     }
 }
 
-/// Returns a modified byte length of the deposit event. Length is calculated with maximum possible
-/// constants for all fields except `asset` and `destination_chain_address`, ergo it may not be
-/// representative of on-wire length. This should always return a byte length greater than or equal
-/// to the actual size of the serialized `Deposit`, but never less.
+/// Returns a modified byte length of the deposit event. Length is calculated with reasonable values
+/// for all fields except `asset` and `destination_chain_address`, ergo it may not be representative
+/// of on-wire length.
 pub(crate) fn get_deposit_byte_len(deposit: &Deposit) -> u128 {
```

Comment:
This should be renamed to give a more "business logic" name. For example `fn calculate_deposit_fee` or something like that.